tests/functional/test_sneddon_2d.py

```diff
@@ -0,0 +1,110 @@
+import copy
+import math
+
+import numpy as np
+import pytest
+
+import porepy as pp
+import tests.functional.setups.manu_sneddon_2d as manu_sneddon_2d
+from porepy.applications.convergence_analysis import ConvergenceAnalysis
+
+# Set up the material constants
+poi = 0.25
+shear_modulus = 1
+lam = (
+    2 * shear_modulus * poi / (1 - 2 * poi)
+)  # Convertion formula from shear modulus and poission to lame lambda parameter
+
+solid = pp.SolidConstants(shear_modulus=shear_modulus, lame_lambda=lam)
+
+
+def compute_frac_pts(
```

Comment:
Back to my comment about the storing `"frac_pts"` in `params`. This function feels like a perfect complement to the Sneddon model setup, because it generates a fracture that satisfies the Sneddon model assumption. The user can pass the parameters to edit the fracture length and rotation etc. I suggest moving it there.

Reply:
I think I don't understand exactly what your argument is. Still, I assume you're right, so if you feel confident, please move ahead. Otherwise, feel free to consult me.